diff --git a/node_modules/serverless/lib/plugins/package/lib/zip-service.js b/node_modules/serverless/lib/plugins/package/lib/zip-service.js
index 94ecfa8..1e28039 100644
--- a/node_modules/serverless/lib/plugins/package/lib/zip-service.js
+++ b/node_modules/serverless/lib/plugins/package/lib/zip-service.js
@@ -5,7 +5,14 @@ const archiver = require('archiver');
 const os = require('os');
 const path = require('path');
 const crypto = require('crypto');
-const fs = BbPromise.promisifyAll(require('fs'));
+
+//const fs = BbPromise.promisifyAll(require('fs'));
+var realFs = require('fs')
+var gracefulFs = require('graceful-fs')
+gracefulFs.gracefulify(realFs)
+const fs = BbPromise.promisifyAll(realFs);
+
+
 const childProcess = BbPromise.promisifyAll(require('child_process'));
 const globby = require('globby');
 const _ = require('lodash');
